solidity/README.md

```diff
@@ -0,0 +1,69 @@
+﻿Solidity contract workflow
+
+Start with source code of the contract in <MyContract.sol>
+
+1) Compile Solidity source to TVM assembler: 
+
+solc --tvm MyContract.sol >MyContract.code
+
+2) Generate public interface for the contract in JSON format: 
+
+solc --tvm_abi MyContract.sol >MyContract.abi.json
+
+3) Assemble and link with libraries of your choice into TVM bytecode: 
+
+tvm_linker compile MyContract.code --lib <path to>stdlib_sol.tvm --abi-json MyContract.abi.json
+
+Binary code of your contract is recorded into <VeryLongNumber.tvc> file.
+Contract is ready to be deployed onto local node or blockchain.
+
+NB: You can test your contracts locally without even starting node:
+
+tvm_linker test --abi-json <MyContract.abi.json> --abi-method <"myMethod"> --abi-params '{"parameter":"value"}' 
+
+This set of smart-contract samples illustrates common functionality of smart-contracts developed in Solidity,
```

Comment:
Comma at the end of line.

Reply:
intended. fixed by de-capitalized "starting"